packages/protvista-variation-adapter/src/tooltipGenerators.ts

```diff
@@ -46,6 +46,20 @@ export const getPopulationFrequencies = (
     )
     .join("")}`;
 
+export const getEnsemblCovidLinks = (variant: Variant) => {
+  const shouldGenerateLink = variant.locations.some(
+    (location) => location.source === "EnsemblViruses"
```

Comment:
Is this always going to be in Ensembl covid 19 portal? "EnsemblViruses" feels like it's going to cover more than just covid 19

Reply:
Yes, confirmed by Jie. Also note the "interesting" way to get the id. 😨

Reply:
Sure... Because why not? 😅  (Similar to what we had at some point in the citations I think, where the pubmeds and doi were xrefs in a list...).